ara/ui/templates/host_index.html

```diff
@@ -114,7 +120,7 @@
           <tr style="height:50px;">
             <th class="text-center" title="Status of the playbook">Playbook</th>
             <th class="text-center" title="Link to the host report">Report</th>
-            <th title="Last update">Last update {% include "partials/sort_by_date.html" with arg="updated" %}</th>
+            <th title="Date at which the host was first recorded">Date {% include "partials/sort_by_date.html" with arg="created" %}</th>
```

Comment:
I see what you mean and I wonder what we should put here.

I came to the realization that relying on the "updated" field is error prone since it gets updated whenever the row changes, including SQL migrations.

How would you describe the created date in a way that is appropriate for both hosts and latesthosts ?

Reply:
no idea. I checked, on the host index improvements I just made "Date" out of it :D 
![image](https://user-images.githubusercontent.com/38468371/189865261-c30ce6e9-4d88-4eb2-951d-9ee26f613d7c.png)

Reply:
I updated it to ``Date this host was recorded`` which is not wrong and probably less confusing.